### PR TITLE
RET-3046 Average weekly hours Decimal Check Fails

### PR DESCRIPTION
**Pull request by JustMehmet**
### JIRA link

https://tools.hmcts.net/jira/browse/RET-3046

### Change description

Decimal value validation is added to average weekly hours field

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Review by hsjhita — APPROVED** [review]

**Review by btensay — APPROVED** [review]

File: src/test/unit/controller/NoticeEndController.test.ts [hunk 1]
```diff
@@ -31,9 +31,9 @@ describe('Notice end Controller', () => {
     const body = {
       'noticeEnds-day': '21',
 
-      'noticeEnds-month': '01',
+      'noticeEnds-month': '12',
 
-      'noticeEnds-year': '2023',
+      'noticeEnds-year': '2030',
```
Comment (GauravChawlaMOJ) [inline src/test/unit/controller/NoticeEndController.test.ts:36]: I think this test was failing before as the date was in past after 21st Jan, worth changing it to year 2777 or something not in our lifetime :-) 7 years seems to pass quickly as we think :)

**Review by GauravChawlaMOJ — CHANGES_REQUESTED** [review]

File: src/test/unit/controller/NoticeEndController.test.ts (hunk 1 above)
Comment (JustMehmet) [reply src/test/unit/controller/NoticeEndController.test.ts:36]: It does not approve too far time Gaurav. Max year that I can enter is 2032. So updated to 2032 :)

**Review by GauravChawlaMOJ — APPROVED** [review]

File: src/test/unit/controller/NoticeEndController.test.ts (hunk 1 above)
Comment (GauravChawlaMOJ) [reply src/test/unit/controller/NoticeEndController.test.ts:36]: ok thanks :)

File: src/main/components/form/validator.ts [hunk 2]
```diff
@@ -119,11 +119,15 @@ export const isValidAvgWeeklyHours: Validator = value => {
...
+  if ((value as string).trim().startsWith('-')) {
+    return 'negativeNumber';
+  }
+
+  if (!/^\d*\.?\d*$/.test(value as string)) {
     return 'notANumber';
   }
 
-  if (((value as string).startsWith('0') && (value as string).trim().length > 1) || (value as string).includes('.')) {
+  if ((value as string).trim().startsWith('.')) {
```
[the remark this answers is not in the document]
Comment (JustMehmet) [reply src/main/components/form/validator.ts:130]: It will be.

[the remark this answers is not in the document]
Comment (JustMehmet) [reply src/main/components/form/validator.ts:130]: Rules are:
Should not contain alphabetic characters.
Can be 0 or 0.5 etc.
Should not have more than 5 characters.
Should only have one '.' character and all other characters should be numeric
Should not start with '.'

**Pull request merged**